cpp/arcticdb/CMakeLists.txt

```diff
@@ -403,6 +404,9 @@ set(arcticdb_srcs
 if(NOT ${ARCTICDB_USING_CONDA})
     list (APPEND arcticdb_srcs
         util/third_party/robin_hood.hpp
+        storage/azure/azure_storage-inl.hpp
+        storage/azure/azure_storage.hpp
+        storage/azure/azure_storage.cpp
```

Comment:
Do we have an issue raised to support Azure in Conda? If so, could link it here.

Reply:
Added